SolidStateGroup/bullet-train-api
# fix: Metadata UI improvements

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Add error messages and validations in the features and environments settings.
- Improve multiline type metadata.
- Change URL validation type.
- Only enterprise Orgs can use custom fields/metadata



## How did you test this code?

- Use an enterprise organization to be able to use custom fields/metadata.
- Changes were applied to multiline metadata in the feature settings.
- Try to cause an error to see error toasts. For example, enter an incorrect metadata value and try to save.
